src/Stratis.Bitcoin.Features.Wallet/WalletFeature.cs

```diff
@@ -193,4 +193,4 @@ public static IFullNodeBuilder UseWallet(this IFullNodeBuilder fullNodeBuilder)
             return fullNodeBuilder;
         }
     }
-}
+}
```

Comment:
Can you please turn off the thing that removes the extra line at the end of files?